yahoo_fantasy_api/league.py

```diff
@@ -638,6 +638,10 @@ def player_stats(self, player_ids, req_type, date=None, season=None):
             what date to request the stats for.  If left as None, and range
             is for a date this returns stats for the current date.
         :type date: datetime.date
+        :param week: NFL ONLY: When requesting stats for a week, this identifies the
```

Comment:
Can you update the comments for the `req_type` parameter for week?  It currently doesn't mention anything about that as being an option.

Reply:
Good call! Thanks for catching that!